sip-core/src/test/java/de/ikor/sip/foundation/core/framework/routers/OnExceptionConfigurationTest.java

```diff
@@ -0,0 +1,77 @@
+package de.ikor.sip.foundation.core.framework.routers;
+
+import de.ikor.sip.foundation.core.apps.framework.CentralRouterTestingApplication;
+import de.ikor.sip.foundation.core.framework.connectors.InConnector;
+import de.ikor.sip.foundation.core.framework.connectors.OutConnector;
+import de.ikor.sip.foundation.core.framework.endpoints.InEndpoint;
+import de.ikor.sip.foundation.core.framework.endpoints.OutEndpoint;
+import de.ikor.sip.foundation.core.framework.stubs.*;
+import org.apache.camel.CamelContext;
+import org.apache.camel.EndpointInject;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.test.spring.junit5.CamelSpringBootTest;
+import org.apache.camel.test.spring.junit5.MockEndpoints;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Description;
+
+@CamelSpringBootTest
+@SpringBootTest(classes = CentralRouterTestingApplication.class)
+@MockEndpoints("log:message*")
+class OnExceptionConfigurationTest {
```

Comment:
Some tests are failing due to order of test execution. By adding @DirtiesContext here they work as expected.